h1st/model/fuzzy_logic/fuzzy_logic_rules.py

```diff
@@ -0,0 +1,100 @@
+import logging
+from typing import NoReturn
+
+import skfuzzy
+from skfuzzy import control as ctrl
+from skfuzzy.control.term import Term
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+class FuzzyLogicRules:
+    def __init__(self) -> None:
+        self.membership = {
```

Comment:
This should be implemented as a CONSTANT, right?

Reply:
I have updated it using enum and also used it in user code. Please review it if this is correct way to do this.